### PR TITLE
Cleanup temporary files during teardown for k8s-copy-files.bats

### PR DESCRIPTION
**Pull request by mingweishih**
Fixes: #5219 

Signed-off-by: Ming-Wei Shih <mishih@microsoft.com>

### CODE REVIEW

File: integration/kubernetes/k8s-copy-file.bats [hunk 1]
```diff
@@ -78,4 +78,6 @@ teardown() {
 
 	rm -f "$file_name"
 	kubectl delete pod "$pod_name"
+
+	rm "$pod_config"
```
Comment (anakrish) [inline integration/kubernetes/k8s-copy-file.bats:82]: Should this be `rm -f` ?

**Comment by liubin** [comment]
Thank you @mingweishih, you need to fix your commit message by [this format](https://github.com/kata-containers/community/blob/main/CONTRIBUTING.md#patch-format).

**Comment by mingweishih** [comment]
> Thank you @mingweishih, you need to fix your commit message by [this format](https://github.com/kata-containers/community/blob/main/CONTRIBUTING.md#patch-format).

Thanks for the pointer. I've updated the commit.

**Review by GabyCT — APPROVED** [review]

**Review by fidencio — APPROVED** [review]
lgtm, thanks @mingweishih!

**Pull request merged**